packages/cli/src/config/world/resolveWorldConfig.ts

```diff
@@ -50,12 +50,14 @@ export function resolveWorldConfig(config: ParsedWorldConfig, existingContracts?
  * @param existingContracts optional list of existing contract names, used to validate system names in the access list. If not provided, no validation is performed.
  * @returns ResolvedSystemConfig object
  * Default value for fileSelector is `systemName`
+ * Default value for registerFunctionSelectors is true
  * Default value for openAccess is true
  * Default value for accessListAddresses is []
  * Default value for accessListSystems is []
  */
 export function resolveSystemConfig(systemName: string, config?: SystemUserConfig, existingContracts?: string[]) {
   const fileSelector = config?.fileSelector ?? systemName;
+  const registerFunctionSelectors = config?.registerFunctionSelectors ?? true;
```

Comment:
wondering about how best to include these in the system call stream:
- figure out the function selectors here and include in the return object?
- wait for the resulting world to compile then use that ABI to determine if its a registered system call?

Reply:
`worldgen` creates a world abi with the correct function selectors (without compiling because it parses solidity directly) - could we use that for they system call stream?